src/main/java/net/mcreator/workspace/Workspace.java

```diff
@@ -496,4 +497,22 @@ public boolean isRegenerateRequired() {
 		this.workspaceSettings.setWorkspace(this);
 	}
 
+	public static boolean isWorkspaceLockedToVersion(File workspaceFile) {

```

Comment:
I have also realized we have WorkspaceUtils so I would move those static methods to this utility class instead